test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -1,403 +1,281 @@
-  chain: ChainName = "Ethereum",
-  cancellation: boolean = true,
-) {
-  const seqAddress = seq.keyRingPair.address;
-  const cancelerAddress = canceler.keyRingPair.address;
-  await Rolldown.waitForReadRights(seqAddress, 50, chain);
-  const txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
-  const api = getApi();
-  const update = new L2Update(api)
-    .withDeposit(txIndex, seqAddress, seqAddress, BN_MILLION)
-    .on(chain)
-    .build();
-  let reqId = 0;
-  await Sudo.asSudoFinalized(
-    Sudo.sudoAsWithAddressString(seqAddress, update),
-  ).then(async (events) => {
-    await waitSudoOperationSuccess(events, "SudoAsDone");
-    reqId = Rolldown.getRequestIdFromEvents(events);
-  });
-  //const disputePeriodStartBlock = await getBlockNumber();
-  let reqIdCanceled: number = 0;
-  if (cancellation === true) {
-    const cancel = await Sudo.asSudoFinalized(
-      Sudo.sudoAsWithAddressString(
-        cancelerAddress,
-        await Rolldown.cancelRequestFromL1(chain, reqId),
-      ),
-    );
-    await waitSudoOperationSuccess(cancel, "SudoAsDone");
-    reqIdCanceled = Rolldown.getRequestIdFromCancelEvent(cancel);
-  }
-  return { txIndex, api, reqId, reqIdCanceled };
-}
-
-const preSetupSequencers = {
-  Ethereum: "0x3cd0a705a2dc65e5b1e1205896baa2be8a07c6e0",
-  Arbitrum: "0x798d4ba9baf0064ec19eb4f0a1a45785ae9d6dfc",
-};
+let chain: any;
+let testUser1: User;
+let testUser2: User;
+let testUser2Address: string;
+let disputePeriodLength: number;
+let providingExtrinsic: any;
 
 beforeAll(async () => {
   await initApi();
-  setupUsers();
   await setupApi();
-  //Add a few tokes because some tests may end up on slashing them
-  await Sudo.batchAsSudoFinalized(
-    Sudo.sudoAsWithAddressString(
-      preSetupSequencers.Ethereum,
-      await SequencerStaking.provideSequencerStaking(BN_ZERO, "Ethereum"),
-    ),
-    Sudo.sudoAsWithAddressString(
-      preSetupSequencers.Arbitrum,
-      await SequencerStaking.provideSequencerStaking(BN_ZERO, "Arbitrum"),
-    ),
-  );
+  disputePeriodLength = (await Rolldown.disputePeriodLength()).toNumber();
 });
 
 beforeEach(async () => {
-  //TODO: Replace this by some monitoring of the active queue.
-  await waitForNBlocks((await Rolldown.disputePeriodLength()).toNumber());
+  //There shouldn't be any sequencer in activeSequencers
+  [testUser1, testUser2] = setupUsers();
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(testUser1),
+    Assets.mintNative(testUser2),
+  );
   const activeSequencers = await SequencerStaking.activeSequencers();
-  let anySequencerGone = false;
   for (const chain in activeSequencers.toHuman()) {
     for (const seq of activeSequencers.toHuman()[chain] as string[]) {
-      if (
-        seq !== preSetupSequencers.Ethereum &&
-        seq !== preSetupSequencers.Arbitrum
-      ) {
-        await leaveSequencingIfAlreadySequencer(seq);
-        anySequencerGone = true;
+      if (seq !== null) {
+        await leaveSequencing(seq);
```

Comment:
is there any reason why you are ejecting the preSetupSequencers?

Reply:
This way I reduced the time it took to complete the tests.